src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -1267,10 +1267,18 @@ private void VisitExpr(ParseNode node)
                 Opcode skipPastFunctionBody = AddOpcode(new OpcodeBranchJump());
                 string functionStartLabel = GetNextLabel(false);
                 
+                needImplicitReturn = true;
                 nextBraceIsFunction = true;
                 VisitNode(node.Nodes[0]); // the braces of the anonymous function and its contents get compiled in-line here.
                 nextBraceIsFunction = false;
-
+                if (needImplicitReturn)
+                    // needImplicitReturn is unconditionally true here, but it's being used anyway so we'll find this block
+                    // of code later when we search for "all the places using needImplicitReturn" and perform a refactor
+                    // of the logic for adding implicit returns.
```

Comment:
Just to confirm, this essentially means that if a function already has a return, this currently just serves as a safety net return statement.  Kind of like this:
```
set h to {
    return 1 + 1. // normal return
    return 0. // safety net that never gets called
}
```